index.js

```diff
@@ -270,14 +318,20 @@ module.exports = function (schema, options) {
             doc.deletedAt = new Date();
         }
 
+
         if (schema.path('deletedBy')) {
             doc.deletedBy = deletedBy;
```

Comment:
🤔 If you ensure `params` is up to date then you should be able to do this
```suggestion
            doc.deletedBy = params.deletedBy;
```